keras_hub/src/models/cspnet/cspnet_presets.py

```diff
@@ -0,0 +1,4 @@
+"""CSPNet preset configurations."""
```

Comment:
can you upload the presets on Kaggle and update this file?

Reply:
Done